lib/modules/datasource/deb/readme.md

```diff
@@ -1,7 +1,9 @@
-The Debian datasource enables Renovate to update packages from Debian repositories. It is ideal for projects that depend on Debian-based systems or distributions. You will need to combine Debian datasource with [regex managers](../../manager/regex/index.md) to update dependencies.
+The Debian datasource enables Renovate to update packages from Debian repositories.
+It is ideal for projects that depend on Debian-based systems or distributions.
```

Comment:
```suggestion
The `debian` datasource is meant for projects that:

- depend on Debian-based systems, or
- depend on Debian-based distributions, like Ubuntu
```

Will the `debian` datasource work with Ubuntu distribution stuff?

- If yes, apply suggestion
- If no: remove the `, like Ubuntu` bit from my suggestion, then apply